### ISSUE TITLE
Different from torch.nn.DataParallel that will separate total batch to each gpu equally?

### ISSUE THREAD

**Issue by hypercost** [vacancy/Synchronized-BatchNorm-PyTorch#23]

**Comment by hypercost** [comment]
I missed to get a instance from DataParallel.
Changing `DataParallelWithCallback(net, gpu_ids)` to `net = DataParallelWithCallback(net, gpu_ids)` worked.
